Resources/Prototypes/DeltaV/Roles/Jobs/Justice/chief_justice.yml

```diff
@@ -0,0 +1,58 @@
+- type: job
+  id: ChiefJustice
+  name: job-name-chiefjustice
+  description: job-description-chiefjustice
+  playTimeTracker: JobChiefJustice
+  antagAdvantage: 6 # DeltaV - Reduced TC: Head of Staff
+  requirements:
+    - !type:RoleTimeRequirement
+      role: JobClerk
+      time: 36000 # DeltaV - 10 hours
+    - !type:RoleTimeRequirement
+      role: JobLawyer
+      time: 36000 # 10 hours
+    - !type:RoleTimeRequirement
+      role: JobProsecutor
+      time: 36000 # 10 hours
+    - !type:OverallPlaytimeRequirement
+      time: 90000 # 25 hours
+    - !type:WhitelistRequirement # whitelist requirement because I don't want any dingus judges
+  weight: 20
+  startingGear: CJGear
+  icon: "JobIconChiefJustice"
+  requireAdminNotify: true
+  supervisors: job-supervisors-captain
```

Comment:
Don't want to lose progress and have to scroll all the way back down but I thought the PR said that CJ is not below the Captain but alongside them?

Reply:
We discussed with the SoP writers on this, Captain has maritime authority over everyone on the vessel no matter who they are. They are still exempt from direct demotion, but otherwise, still need to listen to the captain's orders

Reply:
What makes this PR float or sink, Justice needs to have some sort of authority of Security.

Reply:
> What makes this PR float or sink, Justice needs to have some sort of authority of Security.

Quick answer: They control sentencing and can charge anyone they so desire.
Long answer: View the Google doc linked in the PR description